packages/pyright-internal/src/localization/localize.ts

```diff
@@ -156,16 +174,26 @@ function loadStringsForLocale(locale: string): StringLookupMap {
     // general version.
     const localeSplit = locale.split('-');
     if (localeSplit.length > 0 && localeSplit[0]) {
-        override = loadStringsFromJsonFile(localeSplit[0]);
+        override = loadStringsFromJsonFile(localeSplit[0], microbit);
         if (override !== undefined) {
             return override;
         }
+        // Fallback to English micro:bit strings.
+        if (microbit) {
+            override = loadStringsFromJsonFile('en', microbit);
+            if (override !== undefined) {
+                return override;
+            }
+        }
```

Comment:
Explicit fallback is required for tests, though this might not be the desired behaviour where the translation is not available for another language.

Reply:
Simpler approach in 7468c84f09be08f8eae31002622c67b54a03d755